src/fenics_constitutive/interfaces.py

```diff
@@ -94,15 +94,27 @@ def evaluate(
             history: The history variable(s).
         """
 
-    # Method removed because it is not needed anymore.
-    # @abstractmethod
-    # def update(self) -> None:
```

Comment:
How is that done in an implicit model where the iteration in a single Newton step is performed and then only updated once it is converged?